packages/web3-utils/src/types.ts

```diff
@@ -1,8 +1,24 @@
 export type HexString = string;
 export type ValueTypes = 'address' | 'bool' | 'string' | 'int256' | 'uint256' | 'bytes' | 'bigint';
-export type Address = string;
 export type Bytes = Buffer | Uint8Array | ArrayBuffer | HexString;
 export type Numbers = number | bigint | string | HexString;
+// Hex encoded 32 bytes
+export type HexString32Bytes = HexString;
```

Comment:
All these types should be in web3-common, or along with `eth_execution_api.ts `

Reply:
These are general types or to say particularly type aliases. Not necessary to be used only with Eth API so why these are in `web3-utils`

Reply:
ok, if its not only used in Eth API, still web3-common will be good candidate for holding common types. I think this will make dependency less complicated.